src/odemis/driver/hamamatsurx.py

```diff
@@ -833,9 +845,40 @@ def updateMetadata(self, md):
 
         super(DelayGenerator, self).updateMetadata(md)
 
+    def GetPLLMode(self):
+        """
+        Get the current value from the streak unit HW.
+        :return: (bool) current streak mode value
+        """
+        pllMode_raw = self.parent.DevParamGet(self.location, "PLL Mode")  # returns a list
+        if pllMode_raw[0] == "Locked":
+            pllMode = True
+        elif pllMode_raw[0] == "Unlocked":
+            pllMode = False
+        else:
+            logging.warning("Unexpected phase-locked loop mode %s", pllMode_raw)
+            pllMode = True
+
+        return pllMode
+
+    def _setPLLMode(self, value):
+        """
+        Updates the streakMode VA.
+        :parameter value: (bool) value to be set
```

Comment:
I typed parameter instead of param because all the doc strings were written that way in this module, but let me fix all of them, that would be better :)